ui/src/pages/listing/New.tsx

```diff
@@ -26,37 +27,50 @@ export const New : React.FC = () => {
   const [ quotedInstrumentLabel, setQuotedAssetLabel ] = useState("");
   const [ id, setId ] = useState("");
 
-  const [ showTradedInstrument, setShowTradedAsset ] = useState(false);
-  const [ node, setNode ] = useState<ClaimTreeNode | undefined>();
-
   const { getParty } = useParties();
   const ledger = useLedger();
   const party = useParty();
   const { contracts: services, loading: l1 } = useStreamQueries(Service);
   const { contracts: autoServices, loading: l2 } = useStreamQueries(AutoService);
   const { contracts: instruments, loading: l3 } = useStreamQueries(Instrument);
```

Comment:
You can use the `useInstruments` hook + context here to get access to all instruments. See here: https://github.com/digital-asset/daml-finance-app/blob/main/ui/src/pages/issuance/New.tsx

Reply:
This avoids opening more websocket connections btw.